src/ansible_navigator/utils/serialize.py

```diff
@@ -34,6 +82,17 @@ class JsonParams(NamedTuple):
     ensure_ascii: bool = False
 
 
+def _json_dump(dumpable: Dict, filename: Union[Path, str], file_mode: str):
```

Comment:
Per comment above, always use `Path` internally:
```suggestion
def _json_dump(dumpable: Dict, filename: Path, file_mode: str):
```